giraffe/README.md

```diff
@@ -711,7 +711,45 @@ TableGraphLayerConfig uses the `fluxResponse` property from `config` as the data
 
   - **svgTextStyle**: _Object. Optional. Recommendation: do not include._ An object containing the key-value pairs used for inline styling `.giraffe-single-stat--text` by setting its [style property](https://developer.mozilla.org/en-US/docs/Web/API/ElementCSSInlineStyle/style). If used, please be aware of existing default styles that may need to be overridden. `fill` cannot be overridden and is controlled by the **textColor** option (see above). See the `SINGLE_STAT_SVG_TEXT_DEFAULT_STYLE` [here](./src/constants/singleStatStyles.ts).
 
-* **CustomLayerConfig**: _Object._ No limit per `<Plot>`.
+- **AnnotationLayerConfig**: _Object._ No limit but generally one per `<Plot>`. Properties are:
+
+  - **type**: _'annotation'. **Required**._ Specifies that this LayerConfig is an annotation layer.
+
+  - **x**: _string. **Required**._ The column key name of the column that should be visualized on the x-axis. _Annotations must overlay another data set, ie another graph. Therefore, this column key name is from that data set. This should match the overlaid graph's same property._
+
+  - **y**: _string. **Required**._ The column key name of the column that should be visualized on the y-axis. _Annotations must overlay another data set, ie another graph. Therefore, this column key name is from that data set. This should match the overlaid graph's same property._
+
+  - **fill**: _array[string, ...]. **Required**._ An array of column key names of column filters that should be visualized. _Annotations must overlay another data set, ie another graph. Therefore, these column key names are from that data set. This should match the overlaid graph's same property. If the "fill" is not required in the overlaid graph, please explicitly use the overlaid graph's implicit value in AnnotationLayerConfig._
```

Comment:
what does 'If the "fill" is not required in the overlaid graph, please explicitly use the overlaid graph's implicit value in AnnotationLayerConfig._' mean?  what would an example of the 'implicit value' be?

Reply:
LineLayer, for example, has **fill** as optional. In the README for LineLayer, fill is described as having a default, or implicit value, if fill is not included. There may other graph types with a similar description for fill.

**fill** is a separate, unrelated problem. It really is required, internally, by Giraffe code. Some graph types allow "fill" to be omitted in the layer config as a way of reducing the number of config properties that the user has to think about.

Unfortunately, AnnotationLayerConfig is not one that should allow omitting "fill" because Annotations must overlay something, which makes Annotations a multi-layer layer. Meaning Annotations cannot be used as a single layer. (Even though, visually, Annotations can be rendered as a single layer.)

Reply:
@jrenee42 i will put in an example of the "fill" values in a separate PR that addresses default values.